src/View/TimeframeExport.php

```diff
@@ -96,27 +95,30 @@ public static function exportCsv( string $outputFile = 'php://output' ) {
 			// output the column values
 			$valueColumns = array_values( $timeframeDataRow );
 
+			/** @var \CommonsBooking\Model\Booking $bookingPost */
+			$bookingPost = new \CommonsBooking\Model\Booking( $timeframeDataRow['ID'] );
```

Comment:
Das sind ja nicht nur Bookings die für den Export vorgesehen sind sondern alle Arten von Timeframes, können wir da nicht ein Timeframe Model nehmen? In den Tests scheint es ja auch mit Timeframes zu gehen aber eigentlich kann das ja beides sein.